pxtlib/docsrender.ts

```diff
@@ -418,13 +418,25 @@ namespace pxt.docs {
 
     export function setupRenderer(renderer: marked.Renderer) {
         renderer.image = function (href: string, title: string, text: string) {
-            let out = '<img class="ui image" src="' + href + '" alt="' + text + '"';
-            if (title) {
-                out += ' title="' + title + '"';
+
+            if (href.startsWith("youtube:")) {
+                    let out = '<div class="tutorial-video-embed"><iframe src="https://www.youtube.com/embed/' + href.split(":").pop()
+                        + '" title="' + title + '" frameborder="0" ' + 'allowFullScreen ' + 'allow="autoplay; picture-in-picture"></iframe></div>';
+                    return out;
+
+            } else if (href.includes("youtube")) {
```

Comment:
I missed this when reviewing earlier, but we probably just want to remove this `else if` branch -- it's totally fine to have an image like `/static/my-tutorial/how-to-add-youtube-video-screenshot` .